docs/concepts/publishing.md

```diff
@@ -52,42 +49,34 @@ A published artifact consists of two parts:
 - The artifact metadata
 - The artifact itself
 
-The artifact metadata is uploaded to Flox servers so that the Flox CLI can see that it's available via the `flox search`, `flox show`, and `flox install` commands.
+The artifact metadata is uploaded to Flox servers so that the Flox CLI can see that it's available via the [`flox search`][flox-search], [`flox show`][flox-show], and [`flox install`][flox-install] commands.
 The artifact itself is uploaded to a Catalog Store.
 
-For individual users we provide your Catalog Store.
-For organizations you provide your own Catalog Store, but a hosted offering will be available in the future.
-When the Catalog Store is user-provided it means that your organization has complete control over your artifacts, and many organizations will still choose this route even when a hosted option is available.
+A Catalog Store is effectively a cache for published artifacts, and Flox provides one by default.
+An organization can choose to provide their own Catalog Store in the form of an S3-compatible storage provider.
+In this case, it means that your organization has complete control over your artifacts and they will never be stored by Flox.
+To pursue this option, contact Flox directly.
 
 ## Consuming published artifacts
 
 Once you have uploaded an artifact via `flox publish`, the package becomes available in `flox search`, `flox show`, and `flox install`.
 To distinguish these packages from those provided by the Base Catalog, published packages are prefixed with the name of the user or organization.
-For example, if your user is called `myuser` and it publishes an artifact named `hello`, the artifact will appear as `myuser/hello` in the Flox CLI.
-
-When a user runs `flox install myuser/hello`, the artifact is downloaded directly from the Catalog Store that it was published to.
-
-For individual users, only the user can consume artifacts from their catalog.
-Sharing packages with multiple users is only possible with an organization.
-Access control for users in an organization is set via the `catalog-util` package that is provided to you as part of setting up your organization.
-
-## Configuration
+For example, if your user is called `myuser` and you publish an artifact named `hello`, the artifact will appear as `myuser/hello` in the Flox CLI.
 
-### Catalog Store
+When a user runs `flox install myuser/hello`, the artifact is downloaded directly from the Catalog Store that it was published to, i.e. it is never downloaded to or cached on Flox servers.
```

Comment:
```suggestion
When a user runs `flox install myuser/hello`, the artifact is downloaded directly from the Catalog Store that it was published to. If organizations configure their own Catalog Store it is never downloaded to or cached on Flox servers.
```

Reply:
I'm finding it hard to walk the fine line between "say that BYOC exists
and document it" and "we want to strongly imply that publisher is the default workflow".

Reply:
Fixed